src/app/qgsrulebasedlabelingwidget.cpp

```diff
@@ -626,6 +628,8 @@ QgsLabelingRulePropsWidget::QgsLabelingRulePropsWidget( QgsRuleBasedLabeling::Ru
   connect( mScaleRangeWidget, &QgsScaleRangeWidget::rangeChanged, this, &QgsLabelingRulePropsWidget::widgetChanged );
   connect( groupSettings, &QGroupBox::toggled, this, &QgsLabelingRulePropsWidget::widgetChanged );
   connect( mLabelingGui, &QgsTextFormatWidget::widgetChanged, this, &QgsLabelingRulePropsWidget::widgetChanged );
+  connect( mFilterRadio, &QRadioButton::toggled, this, [ = ]( bool toggled ) { filterFrame->setEnabled( toggled ) ; } );
+  connect( mElseRadio, &QRadioButton::toggled, this, [ = ] { editFilter->setText( QStringLiteral( "ELSE" ) );} );
```

Comment:
I think you want

    connect( mElseRadio, &QRadioButton::toggled, this, [ = ]( bool toggled ) {  if ( togggled ) editFilter->setText( QStringLiteral( "ELSE" ) );} );